pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -467,7 +506,7 @@ func (facts *PipelineRunFacts) GetPipelineConditionStatus(ctx context.Context, p
 		}
 
 		switch {
-		case s.Failed > 0:
+		case s.Failed > 0 || s.SkippedDueToTimeout > 0:
```

Comment:
we need to update the `message` added to the pipelinerun status to include `SkippedDueToTimeout`

https://github.com/tektoncd/pipeline/blob/ef73cfecafaf5b3870f5b3c7d2b83f99c73d50fd/pkg/reconciler/pipelinerun/resources/pipelinerunstate.go#L462-L463

Reply:
I deliberately didn't do that - they _are_ skipped, so I think it makes sense to just leave them in that count. `SkippedDueToTimeout` is a subset of `Skipped` with the sole purpose of ensuring that the `PipelineRun` as a whole is marked as failed, but the actual tasks are just skipped.

Reply:
aah I'd missed that you increment `skipped` whenever you increment `skippedduetotimeout`, sounds good!